content/work.rst

```diff
@@ -194,14 +193,14 @@ Well documented, small tasks based work
 It feels flattering to be essential to the project.
 But that usually also means that project won’t make rapid progress without you.
 
-This is not a good thing. Both for you and the project.
-What happens if want to take a (spontaneous) vacation or get sick?
-Or some work gets delayed and overlaps with your great vacation, scheduled many months before.
+This is not a good thing. Both for you and for the project.
+What happens if want to take a (spontaneous) vacation or if you get sick?
```

Comment:
Add _you_ in this line: _What happens if you want to take a (spontaneous) vacation or if you get sick?_